package.json

```diff
@@ -38,7 +38,7 @@
     "pretest": "eslint .",
     "lint": "eslint .",
     "lint-fix": "eslint . --fix",
-    "test": "mocha --timeout 20000 --slow 0 --reporter spec test/*.spec.js"
+    "test": "mocha --timeout 20000 --slow 0 --reporter spec"
```

Comment:
Need to keep this, and modify the package.json with:
```
"test:unit": "mocha --timeout 20000 --slow 0 --reporter spec"
"test": "npm run test:unit -- test/*.spec.js"
```

It's for me ;) Thanks @deepu105